new_platforms/vscode-extension/README.md

```diff
@@ -106,7 +106,7 @@ the host and enclave symbols into an instance of the debugger.
     
 * Ensure that the requirements are met for the [Azure IoT Edge extension](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.azure-iot-edge):
     * [Docker is installed and running: https://docs.docker.com/install/](https://docs.docker.com/install/).
-        * On Linux, to cross-build a container (i.e. if your host is amd64 and your container is arm32v7), you will need to enable cross-building.  One solution is to add these packages:
+        * On Linux, to cross-build a container (i.e. if your host is amd64 and your container is arm32v7), you will need to enable cross-building.  Adding these packages solved the cross-build issue on Ubuntu 19.04:
```

Comment:
Suggest rephrasing to not say something is "solved" or that there's an "issue", e.g.,  "To enable cross-building on Ubuntu 19.04, do the following:"